test/FsAutoComplete.Tests.Lsp/CoreTests.fs

```diff
@@ -330,9 +330,9 @@ let tooltipTests state =
             13
             (concatLines
               [ "val inline add:"
-                "   x: ^a (requires static member ( + ) ) ->"
-                "   y: ^b (requires static member ( + ) )"
-                "   -> ^c" ])
+                "   x: 'a (requires static member ( + ) ) ->"
```

Comment:
hat types (^) are no longer required, so we should standardize onto normal backticks.